### ISSUE TITLE
Annotations of Rho GTPase cycle pathway members derived from Reactome

### ISSUE THREAD

**Issue by cmungall** [geneontology/pathways2GO#85]
Please don't feel the need to comment on this immediately, I am still collecting some thoughts here.

I am looking at Rho GTPase cycle
http://noctua-dev.berkeleybop.org/editor/graph/gomodel:R-HSA-194840

How close is this to how a GO editor would model this?

(of course we know there are differences in R vs G view, so close this ticket if not helpful, but I feel there could be more here)

It feels like we should be able to derive from MF annotations and a more GO-CAMesque view here... and it seems an interesting one to model due to the cycle... but I found myself getting a bit diverted when looking at existing terms and annotations in GO. E.g

 * https://github.com/geneontology/go-ontology/issues/18749 - NGEF and relationship to GTPase...
 * https://github.com/geneontology/go-annotation/issues/2793 - GTPase activators annotated as GTPases...

I will write more here later...

**Comment by goodb** [comment]
@cmungall no specific comment here, just noting that we've had quite a bit of discussion about ways to make things look more like other GO-CAMs and ways to leverage existing annotations to produce better, more complete models.  We are now discussing these in the context of 'phase 2' of this project and beginning to collate ideas under a new project https://github.com/geneontology/pathways2GO/projects/2 

I am aiming for phase 1 to complete with a very high precision, automated conversion.  For the next iteration, I think making larger leaps of inference (and dealing somehow with the high likelihood that these will come with errors) will make sense.  

As you probably know, what I would really love to see would be objective, automated ways to evaluate improvements from such inferences.  If we make the model more GO-CAMesque, why is it better and by how much?

**Comment by ukemi** [comment]
It seems the key elements to discuss in this model have been issues that creep up continuously. What is 'in' the pathway and what regulates it. This is actually a good example of where we can harmonize with Reactome in the phase 2 part of this project. Off the top of my head, I would draw the boundary of the pathway such that the GAP negatively regulated the activity of the GTP-bound GTPase and thus negatively regulated the pathway. But this is one of those cases, like Wnt signaling, where you almost never see the pathway without the GAP. So the decision we need to make from the biology side is do we stick with the current view where the GAP is included in the representation of the pathway and has a negative effect on the GTP-bound GTPase or do we split it out as a negative regulator? We will work this out in conjunction with Reactome. I think at the end of the day it's subjective and we look to see what the community expects. Do we expect to get the GAPs when we query for genes that are involved in Ras signaling? If we do, then we keep it as is. If we don't we split it out as regulatory. We need to stay practical. @deustp01 @vanaukenk

**Comment by deustp01** [comment]
We are starting a project to revise / expand our annotations of RHO proteins and RHO-specific GEFs and GAFs, so I'm passing this on to the people directly involved - Marija Milacic, Karen Rothfels, and Lisa Matthews.

**Comment by cmungall** [comment]
Thanks

@goodb - the 2 phase plan makes perfect sense, cool. I have been aware of the ongoing discussions re how native the CAM representation should be, I found looking in more detail at this example gave me some additional insight but you are probably all way ahead of me

@ukemi - I wasn't so concerned with the part-of links and defining start/end (though this is important for the derived GPADs and other applications).  was more looking at causal connections between activities

@deustp01 - you mean GAPs or GAFs? :smile: so are you talking about annotating specific GEFs, GAPs, and GTPases? I am sure there has already been discussion about how we go from  REACTO groupings to specific gene products that people expect to see in GAFs, for enrichment etc. Obv not all combos are possible...

**Comment by deustp01** [comment]
That was a gaf(fe) - sorry - GAPs (GTPase activating proteins) meant, and while on our side I'm sure the new work will generate still more sets of gene products) they will still all be intended to group assertions about the shared functions of tyhe members of families of proteins.

**Comment by ukemi** [comment]
This model and the Reactome representation might be worth discussing if you have a call next week. I will not be available.

**Comment by ukemi** [comment]
@cmungall I don't know the GDI story well enough, but given the instance issue of a single RhoGTPase:GTP going on to two different downstream reactions, I think this looks ok to me. What is an instance of the RhoGTPase:GTP? We have allowed this kind of split so far, but as many know it still makes me uneasy in my reductionist view of the world. Note that in the original Reactome model, the GAP reaction doesn't negatively regulate the activity of the effector complex. I think we have accomplished what we set out to do, accurately reflect the Reactome view in GO-CAM. Phase 2 is where the real fun starts and we get down to discussing weedy biology.

**Comment by deustp01** [comment]
> I am sure there has already been discussion about how we go from REACTO groupings to specific gene products that people expect to see in GAFs, for enrichment etc. Obv not all combos are possible...

Right, and in our RHO annotations so far we have probably done more than the usual amount of lumping gene products into sets in ways that obscure true functional distinctions - an issue both for our planned cleanup and for looking for good Reactome test cases for the REACTOME - GO-CAM mapping
